servicecontrol/servicecontrol-instances/distributed-instances.md

```diff
@@ -161,7 +166,11 @@ Configure slave 1 to be the new master, enable error
 Shutdown master
 Point tools to Slave 1 which is the new master
 
-#### Multi-region handling
+### Multi-region deployments
+
+Mutli-region deployments are partially supported (error data sharding is not supported). Such setup consists of ServiceControl slave deployed in each region and a master instance responsible for aggregating audit data. 
```

Comment:
Why remove these two paras?

Reply:
The changes to this part are the results of saying "multi-region is partially suported" vs. "this is how we support multi-region".